src/renderer.cpp

```diff
@@ -596,25 +625,173 @@ void Renderer::shutdown()
 
     bgfx::shutdown();
 }
+void Renderer::clearShadowMap()
+{
+    // Save current view id
+    int tmpBackendViewId = backendViewId;
 
-void Renderer::draw()
+    backendViewId = 0;
+    setClearFlags(false, true);
+
+    // Revert view id to its original value
+    backendViewId = tmpBackendViewId;

```

Comment:
Same for this one

Reply:
Removed the need for this change, see comment above.